classifiers/premiums/gpt3_classifier/code_snippet.md

```diff
@@ -26,6 +26,8 @@ def gpt3_classifier(record):
     - presence_penalty: Value between -2.0 and 2.0. Positive values penalize new tokens based on whether they appear in the text so far, increasing the model's likelihood to talk about new topics.
     """
     # Access openai via API key
+    if not record[YOUR_ATTRIBUTE] or not record[YOUR_ATTRIBUTE].text:
+        return "No text string read!"
```

Comment:
doesnt work -- this will run into errors since we don't have a label "No text string read!"

Reply:
printing & returning (without text) should do it. Did no one test this?

Reply:
also it doesn't work regardint the access 
![image](https://user-images.githubusercontent.com/91723236/219389905-fd127e61-f947-4dcb-af02-1fa053bc1494.png)

Reply:
I tried to test it on a sample project with adding some empty text, but might have been some mix up because it bypassed that clause and gave no error. Will fix this as soon as possible :)